101-signalr/azuredeploy.json

```diff
@@ -54,11 +54,37 @@
       "metadata": {
         "description": "Visit https://github.com/Azure/azure-signalr/blob/dev/docs/faq.md#service-mode to understand SignalR Service Mode."
       }
+    },
+    "enableConnectivityLogs": {
+      "type": "string",
+      "defaultValue": "true",
+      "allowedValues": [
+        "true",
+        "false"
+      ]
+    },
+    "enableMessagingLogs": {
+      "type": "string",
+      "defaultValue": "true",
+      "allowedValues": [
+        "true",
+        "false"
+      ]
+    },
+    "allowedOrigins": {
+      "type": "array",
+      "defaultValue": [
+        "https://foo.com",
+        "https://bar.com"
+      ]
     }
   },
+  "variables": {
+    "signalRApiVersion": "2020-05-01"
```

Comment:
need to remove this now...  

also check the CI results for the other changes needed.